apps/user-profile/src/components/modules/UserOnboardingModal/UserOnboardingModal.js

```diff
@@ -13,6 +15,13 @@ Modal.setAppElement("#__next");
  */
 export default function UserOnboardingModal({ isOpen }) {
     const [modalIsOpen, setModalIsOpen] = useState(true);
+    const { userData } = useUserDataContext;
+
+    let percentage = () => {
+        // hasDone(task) has not yet been created but the implementation will be as follow
+        const task = [userData.hasDonePlatformOnboarding, userData.hasDoneDiscordOnboarding, userData.hasDoneThirdPartyOnboarding];
```

Comment:
have the const variable `task` outside the percentage function.  You can then use it to dynamically set the "1/4" tasks number e.g `{taskDone/task.length}

Reply:
I have commented that part of the code({`taskDone/task.length`} logic) at the end of the line in order to avoid that runtime error
![2023-05-08](https://user-images.githubusercontent.com/110208240/236924990-e4e79df2-c65b-4fae-bf11-1ed4cfdca4b9.png)